server/chat/backend/agent/tools/alert_payload_tool.py

```
@@ -0,0 +1,155 @@
+        )
+    )
+
+
+GET_ALERT_FIELD_DESCRIPTION = (
+    "Retrieve the full (untruncated) value of a field from the original webhook payload. "
+    "Use when the RCA prompt shows a truncated field you need to inspect fully. "
+    "Provide a dot-separated JSON path (e.g. 'event.incident.custom_field_entries', 'alerts.0.annotations')."
+)
+
+
+def get_alert_field(
```

Ruff (0.15.14)

[warning] Too many return statements (13 > 6)

(PLR0911)